### DISCUSSION TITLE
How to implement Bar Chart Race (Racing bar chart) Animation in Visx?

### DISCUSSION THREAD

**Discussion by toughyear** [airbnb/visx#1223] [General]
How to implement this type of animation using Visx?

This might a simple question, but I tried to search for this issue but couldn't find anything.

Here is an example from D3.js.
https://observablehq.com/@d3/bar-chart-race

**Comment by williaster** [comment]
Hey @toughyear I'm going to convert this to a discussion since it's more of a usage question than a `visx` issue. 

Overall I haven't seen this specifically implemented, but most of the work for animating the bars would be outside of `visx`, probably with `react-spring` for performance and using `@visx/scale` or `d3-scale` for positioning math. And then you'd need to update the `visx` axes and grid (from `@visx/react-spring`?), and scales in quick succession.
